raw_data.php

```diff
@@ -1,34 +1,35 @@
 <?php
+
 include('config/config.php');
-include('utils.php');
+
 
 $now = new DateTime();
 
 $d = array();
 
 $d["timestamp"] = $now->getTimestamp();
 
-$swLat = isset($_GET['swLat']) ? $_GET['swLat'] : 0;
-$neLng = isset($_GET['neLng']) ? $_GET['neLng'] : 0;
-$swLng = isset($_GET['swLng']) ? $_GET['swLng'] : 0;
-$neLat = isset($_GET['neLat']) ? $_GET['neLat'] : 0;
-$oSwLat = isset($_GET['oSwLat']) ? $_GET['oSwLat'] : 0;
-$oSwLng = isset($_GET['oSwLng']) ? $_GET['oSwLng'] : 0;
-$oNeLat = isset($_GET['oNeLat']) ? $_GET['oNeLat'] : 0;
-$oNeLng = isset($_GET['oNeLng']) ? $_GET['oNeLng'] : 0;
-$luredonly = isset($_GET['luredonly']) ? $_GET['luredonly'] : false;
-$lastpokemon = isset($_GET['lastpokemon']) ? $_GET['lastpokemon'] : false;
-$lastgyms = isset($_GET['lastgyms']) ? $_GET['lastgyms'] : false;
-$lastpokestops = isset($_GET['lastpokestops']) ? $_GET['lastpokestops'] : false;
-$lastlocs = isset($_GET['lastslocs']) ? $_GET['lastslocs'] : false;
-$lastspawns = isset($_GET['lastspawns']) ? $_GET['lastspawns'] : false;
-$d["lastpokestops"] = isset($_GET['pokestops']) ? $_GET['pokestops'] : false;
-$d["lastgyms"] = isset($_GET['gyms']) ? $_GET['gyms'] : false;
-$d["lastslocs"] = isset($_GET['scanned']) ? $_GET['scanned'] : false;
-$d["lastspawns"] = isset($_GET['spawnpoints']) ? $_GET['spawnpoints'] : false;
-$d["lastpokemon"] = isset($_GET['pokemon']) ? $_GET['pokemon'] : false;
-
-$timestamp = isset($_GET['timestamp']) ? $_GET['timestamp'] : 0;
+$swLat = isset($_POST['swLat']) ? $_POST['swLat'] : 0;
+$neLng = isset($_POST['neLng']) ? $_POST['neLng'] : 0;
+$swLng = isset($_POST['swLng']) ? $_POST['swLng'] : 0;
+$neLat = isset($_POST['neLat']) ? $_POST['neLat'] : 0;
```

Comment:
Consider adding a minimum of required fields, such as these 4 for the viewport, otherwise it's too easy to abuse. Making sure the viewport is a maximum of distance (or using `LIMIT` in your queries to limit the number of rows + using sorting by distance to the middle of the viewport helps a lot to reduce the load).